tests/test_options.py

```diff
@@ -4,6 +4,7 @@
 import pytest
 
 import click
+from click import Option
```

Comment:
Use `click.Option` below.

Reply:
Okay, but note that using `click.Option` will require 2x the number of lines because of the line width limit.

Reply:
Ah, never mind then, import is not a big deal.